helm/prometheus-rules/templates/alerting-rules/kiam.rules.yml

```diff
@@ -31,7 +31,7 @@ spec:
       annotations:
         description: '{{`Kiam pod {{ $labels.namespace}}/{{ $labels.pod_name }} on {{ $labels.cluster_id}}/{{ $labels.cluster }} has increased STS issuing errors.`}}'
         opsrecipe: kiam_sts_issuing_errors_total/
-      expr: increase(kiam_sts_issuing_errors_total[10m]) > 5
+      expr: rate(kiam_sts_issuing_errors_total[15m]) > 0.10
```

Comment:
converting it to `rate` and only get an alert when the amount of errors jumps by 10% withing 15min. does this sounds feasible? WDYT

Reply:
Sounds reasonable to me. It is a bit hard to test it but we could see how it turns out in practice.